README.md

```diff
@@ -123,28 +139,34 @@ mvn package
 However, for general usage, pulling a pre-built jar from maven is
 recommended.
 
-If you would like to run SnowflakeIngestBasicExample.java or SnowflakeStreamingIngestExample.java in the example folder, 
+If you would like to run SnowflakeIngestBasicExample.java or SnowflakeStreamingIngestExample.java in the example folder,
 please edit `pom.xml` and change the scope of the dependency `slf4j-simple` from `test` to `runtime` in order to enable
-console log output.
-
+console log output. For changing the logging level or message format, updating the system property or a
+simplelogger.properties file on the classpath is required, please
+see https://www.slf4j.org/api/org/slf4j/simple/SimpleLogger.html for more details. If you run into formatting errors,
+please run format.sh to format the code.
```

Comment:
This is the major changes, all other changes in this file are done by the auto formatter